### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by sulaimonaa**
Write queries for the following quests:
Find all animals whose name ends in "mon".
List the names of all animals born between 2016 and 2019.
List the names of all neutered animals with less than 3 escape attempts.
List the date of birth of all animals named either "Agumon" or "Pikachu".
List name and escape attempts of animals that weigh more than 10.5kg
Find all animals that are neutered.
Find all animals not named Gabumon.
Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equal
animals-name-argumon-pikachu
animals-neutered-escape-attempts
select-animals-between-2016-2019
animals-name-ends-mon
insert-into-table
 precisely 10.4kg or 17.3kg)

create-table-animals
animals-weight-10 4-17 3
animals-gabumon
animals-neutered-true
animals-weight-10 5

### CODE REVIEW

File: data.sql
Comment (Efiamotu-1) [inline data.sql:1]: [OPTIONAL]
- [ ] I would advice that you make the *id column not null and autoincremented*, in your sche,ma.sql file, see the below snippet as a guide
`id int NOT NULL IDENTITY(1,1)`

File: queries.sql [hunk 1]
```diff
@@ -2,9 +2,10 @@
 
 SELECT * from animals WHERE name LIKE '%mon';
 
-SELECT name FROM animals WHERE YEAR(date_of_birth) BETWEEN 2016 AND 2019;
+SELECT name FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
 
-SELECT name FROM animals WHERE neutered = 1 AND escape_attempts < 3;
```
Comment (Efiamotu-1) [inline queries.sql:7]: - [ ] This query should be written as the code below. currently what you have is `neutered = 1`
```sql
SELECT * FROM animals WHERE neutered = 'true';
```

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @sulaimonaa, 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by shahadat3669 — APPROVED** [review]
Hi @sulaimonaa, 

Your project is complete!  There is nothing else to say other than it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ put correct sql commands in the correct files
✅ create table inside called animals with required columns
✅ add the required datas in the data.sql file

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
- [Optional] the branch name you have chosen does not provide a clear indication of the feature or action being implemented. It is important to use descriptive branch names that accurately describe the purpose of the branch. For example, instead of a generic name like "feature-branch," consider using names like "**create_animals_table**" or "**add_animals_table**" that convey the specific feature or action being worked on. :pray: Please rename the branch to a meaningful name. Check out this [official document](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/renaming-a-branch) for further help. :+1:

Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table  width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadata3669">
        GitHub
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        LinkedIn
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        Slack
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        Email
      </a>
    </td>
  </tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**